pallets/Assets/src/lib.rs

```diff
@@ -90,5 +92,19 @@ decl_module! {
 			})
 		})
 		}
+
+		/// Withdraw
+		#[weight = 1000]
+		pub fn withdraw(origin, dest_id: ChainId, resource_id: ResourceId, to: Vec<u8>, amount: u128) -> DispatchResult {
```

Comment:
```suggestion
		pub fn withdraw(origin, dest_id: ChainId, resource_id: ResourceId, to: Vec<u8>, amount: T::Balance) -> DispatchResult {
```
Shouldn't this be like this?